.travis.yml

```diff
@@ -91,6 +91,10 @@ jobs:
       name: 'Local Tests'
       script:
         - unbuffer node build-system/pr-check/local-tests.js
+    - stage: test
```

Comment:
Should remote tests be removed here?

Reply:
I added a separate job because I figured we could keep an eye on remote-tests.

Reply:
Maybe we can disable them on PR/master builds and only enable them on nightly/branch builds? I worry that leaving it in adds a lot of red/noise to the day-to-day, especially given that we have no idea how long it'll be borked

Reply:
I don't think remote tests should be removed because there is value in running them. Currently, they do not fail PR or push builds, so it shouldn't add any extra redness. 

https://github.com/ampproject/amphtml/blob/947e7f31f577478a78793b9464035a428bd9457b/.travis.yml#L127-L129

Reply:
Even though it doesn't block PRs, it still turns the Travis build red, so the build cop will have to visit every build to see if it's a real failure or not. Are we able to let it fail without turning the build red on Travis? I suspect this may have downstream effects on release automation that promotes the last green Travis build (and we don't have a single green build so far today)

Reply:
Can you point to a build that is red because of `remote-tests.js`? The `allow_failures` section should prevent that, AFAIK.

Reply:
I see green builds with failing Sauce Labs tests, so you're right; I was misreading how Travis treats job/build status